superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -32,6 +32,7 @@ import ColumnElement from './ColumnElement';
 import ShowSQL from './ShowSQL';
 import ModalTrigger from '../../components/ModalTrigger';
 import Loading from '../../components/Loading';
+import { RightOutlined } from '@ant-design/icons';
```

Comment:
Hey, we actually use this for our Icons:
https://github.com/apache/superset/blob/master/superset-frontend/src/components/Icon/index.tsx

THis is a place where we use it which might help:
https://github.com/apache/superset/blob/master/superset-frontend/src/views/CRUD/data/query/QueryList.tsx

Reply:
I see, thank you I will update that as soon as I can!